framework/src/partitioner/PetscExternalPartitioner.C

```diff
@@ -96,8 +96,8 @@ PetscExternalPartitioner::partition(MeshBase & mesh, const unsigned int n_parts)
   if ((mesh.n_elem() / n_parts < 28) && _part_package == "parmetis")
   {
     Moose::out
-        << "Number of elements per compute node must be less than 28, otherwise we switch to "
-           "PTScotch"
+        << "Average number of elements per compute node (" << mesh.n_elem() / n_parts
```

Comment:
```suggestion
        << "Average number of elements per partition (" << mesh.n_elem() / n_parts
```

Reply:
When I think of compute node, I think of HPC

Reply:
same. And we could have threads or have multiple mpi ranks per node. anyway fixed now